apps/aecore/src/aec_governance.erl

```diff
@@ -95,8 +97,8 @@ block_mine_reward() ->
 block_gas_limit() ->
     application:get_env(aecore, block_gas_limit, ?BLOCK_GAS_LIMIT).
 
-tx_base_gas(contract_call_tx) -> 5 * ?TX_BASE_GAS;
-tx_base_gas(contract_create_tx) -> 4 * ?TX_BASE_GAS;
+tx_base_gas(contract_call_tx) ->  30 * ?TX_BASE_GAS;
```

Comment:
Wow, that is steep... But if that is how long it takes to run a contract...

Reply:
Yes, steep, but for a call to idle function compared to normal spend... a reasonable factor

Reply:
Based on a normal spend compared to a contract call to the idle function

Reply:
OK, we are working on improving this ;-)